### ISSUE TITLE
Work View shows all work types 

### ISSUE THREAD

**Issue by julesies** [projecthydra/sufia#3136]
This is maybe a feature request, but I wanted to make sure it's known. Sorry for noise, if so.

We are just now testing multiple work types in 7.3.0rc2. We have two Work, Dataset. I noticed on the work view, the top section "Relationships" is growing and it's really not meaningful if there are no relationships. If say we have 5 work types...would they all show? 

![screen shot 2017-03-10 at 9 05 02 am](https://cloud.githubusercontent.com/assets/4356335/23801980/e3836c96-057f-11e7-9d55-210a8ae80db8.png)

**Comment by mjgiarlo** [comment]
@julesies makes sense to me. Putting this on the backlog and copying the issue to Hyrax too, where I will enlist the help of our UI/UX group.
